Changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.1.0.1 (Jul 2021)
+
+* Move the executable dependencies under the if statement
```

Comment:
* Workaround to avoid incorrect display of dependencies on Hackage by moving build-depends of ucd2haskell executable under a build flag conditional.